README.md

```diff
@@ -0,0 +1,132 @@
+# TABObserverSet
+
+`TABObserverSet`, originally conceived by [Mike Ash](https://github.com/mikeash/SwiftObserverSet), provides a Swift-y alternative to the traditional `NotificationCenter` style of reactive programming.
```

Comment:
Umm well it doesn't use it under the hood if that's what you're asking? I mean it's probably doing the same thing as NC does internally, but it doesn't _use_ NC